src/teaching-element/Assessment/index.vue

```diff
@@ -155,6 +157,18 @@ export default {
     submissionPayload() {
       const { id, userAnswer: answer, isReflection, isCorrect: correct } = this;
       return { data: { id, answer }, isReflection, correct };
+    },
+    parsedQuestion() {
+      if (this.type !== 'FB') return this.question;
+      const counter = { val: 0 };
+      const blankReplacer = buildBlankReplacer(counter);
+      return this.question.map(({ id, data, type }) => {
+        if (!type.includes('HTML')) return { id, data, type };
+        const prevTotal = counter.val;
+        const content = data.content.replace(/@blank/g, blankReplacer);
```

Comment:
How about converting it to blank processor? First there is blank processor factory that optionally recieves target regex and then you pass content to newly created processor in order to get final text.

Reply:
I have been trying for a little while to come up with a clean/elegant way of doing that and having the counter functionality and am drawing a blank (no pun intended).  I am probably missing something obvious. But we are in a time crunch to get this out to fix a broken deploy. So, if you do not see any other issues, we leave this for another PR?